src/components/navbar/Navbar.js

```diff
@@ -44,6 +66,14 @@ const Navbar = ({ clickedLogin, clickedSignup, logOut }) => (
           </Menu>
         ) : (
           <Menu mode="horizontal" style={{ float: 'right' }} data-test="unauthenticated-menu">
+            <Item className="modified-item">
+              <Input.Search
+                placeholder="Search"
+                onSearch={value => search(searchFilter, value, history)}
```

Comment:
You should invoke search action `iff` value has data. e.g:
value => value && search(searchFilter, value, history)

Reply:
Done